lib/get-dependency-examples/utils.js

```diff
@@ -10,8 +10,9 @@ import { EXAMPLE_DIRECTORIES, MODULES } from '../consts';
  * @returns {Array} A list of dependency example paths
  */
 export const getExampleDirectoryPaths = function (dependencySetup) {
-  return EXAMPLE_DIRECTORIES
-    .map((exampleDirectory) => path.join(MODULES, dependencySetup.toLocalName(), exampleDirectory));
+  return EXAMPLE_DIRECTORIES.map(
```

Comment:
Why format map like this? If you need to chain something to it, it wouldn't be readable